Grammatical Fixes in DiskOS's help files.

Cheezed opened the pull request:
**Type:** Typo Fix

**Description:** This revises all of the DiskOS help files, making the grammar slightly better.

**Modified Section:** DiskOS

Commit:
Grammar in all of the help files.

Specifically the DiskOS folder.

Commit:
Final fix.

Commit:
Update Editors

I was being dumb, and forgot a comma.

boredom101 commented:
You should replace it with [https://liko-12.github.io/Disks/](https://liko-12.github.io/Disks/).
Thank you.

Commit:
Fixed link

https://liko-12.github.io/Disks/

Also, removed a space between a word and exclamation mark.

boredom101 reviewed (dismissed):
This looks very good, I appreciate you doing the tedious but important task of correcting grammar.

ghost commented:
Thanks for the feedback! If I catch anything else, I'll make sure to update this.

Commit:
A nation of punctuation!

Commit:
Graves should not be used in suppliment of apostrophes.

pablomayobre commented on OS/DiskOS/Help/API_2:
```diff
@@ -1,7 +1,7 @@
 \CAPI V2 - Since \LIKO-12\C V1.0.0\7
 
-\91.\7 The extra libraries are no longer available as globals, but instead are now loadable by calling `local lib = Library("library_name")`.
+\91.\7 Since the extra libraries are no longer available as globals, they are now loadable by calling 'local lib = Library("library_name")'.
```
Backticks are used to properly format code in Markdown (which LIKO-12 uses for docs)

\`local\` becomes `local`

Honestly I'm not sure if this should remain with backticks (which would translate directly to markdown and later to the JSON format that will be used for the [API docs](https://github.com/LIKO-12/API-Documentation)) or if we want to replace it to just ticks which display correctly in LIKO-12.

I'd prefer the later since the docs will probably need to be rewritten to fit the JSON format anyway.

@boredom101 what do you think?

pablomayobre commented on OS/DiskOS/Help/Tips:
```diff
@@ -1,23 +1,23 @@
-A backup of the .lk12 is created when saving at C:/_backup.lk12
\ No newline at end of file
+Did you know that there's a public GitHub repository for LIKO-12 disks? Check them out at 'https://liko-12.github.io/Disks/'.
+LIKO-12 can use post shaders, check 'https://github.com/RamiLego4Game/LIKO-12-Shaders'.
```
This link should be `https://github.com/LIKO-12/Shaders`

pablomayobre commented on OS/DiskOS/Help/Tips:
```diff
@@ -1,23 +1,23 @@
\ No newline at end of file
+Did you know that there's a public GitHub repository for LIKO-12 disks? Check them out at 'https://liko-12.github.io/Disks/'.
+LIKO-12 can use post shaders, check 'https://github.com/RamiLego4Game/LIKO-12-Shaders'.
+Don't forget to give a star on the LIKO-12 GitHub page! Visit 'https://github.com/RamiLego4Game/LIKO-12'.
```
And this one should be `https://github.com/LIKO-12/LIKO-12`

pablomayobre commented on OS/DiskOS/Help/Tips:
```diff
@@ -1,23 +1,23 @@
+Did you know that there's a public GitHub repository for LIKO-12 disks? Check them out at 'https://liko-12.github.io/Disks/'.
+LIKO-12 can use post shaders, check 'https://github.com/RamiLego4Game/LIKO-12-Shaders'.
+Don't forget to give a star on the LIKO-12 GitHub page! Visit 'https://github.com/RamiLego4Game/LIKO-12'.
+LIKO-12 Nightly-Builds are available at 'https://ramilego4game.github.io/LIKO-12-Nightly'.
```
This is now available at `https://liko-12.github.io/Nightly/`

pablomayobre reviewed (approved):
I approve this PR, I'll wait for the comments on whether to use backticks or ticks and after that's done I'll merge even if the links aren't fixed by then (I can just commit those later)

pablomayobre reviewed (changes requested):
Okey, I asked Rami and the backticks are used to highlight the words, so we want to use backticks for the highlighted parts and not ticks...

ghost commented:
Oh, ok. That’s understandable. I’ll revert it back.

Commit:
Graves SHOULD be used in suppliment of apostrophes.

Commit:
Link-a-dink!
